menas/src/test/resources/log4j.properties

```diff
@@ -22,3 +22,6 @@ log4j.appender.general.layout=org.apache.log4j.PatternLayout
 log4j.appender.general.layout.ConversionPattern=@log.pattern@
 
 log4j.logger.menas.services.RunService=@log.level.testcode@
+
+# (useful) uncomment to see spring beans wiring and related information:
+# log4j.logger.org.springframework.beans.factory.support.DefaultListableBeanFactory=DEBUG
```

Comment:
Please add a new line here and in IntelliJ check that you have an editorconfig plugin and that it is enabled.

Reply:
Aha! I had both, but that was not all that was needed. The `.editorconfig` file was missing a specific setting for `.properties` file type. Added:
```
[*.properties]
insert_final_newline = true
```